tests/router/test_router_e2e_with_mockers.py

```
@@ -368,6 +368,125 @@ async def send_request_via_python_kv_router(
+        # Dump KV events from the router
+        events_json = await router.dump_events()
+
+        # Parse the JSON events and extract worker IDs
+        events = json.loads(events_json)
+        event_worker_ids = sorted({e["worker_id"] for e in events})
+        logger.info(f"Discovered worker IDs: {event_worker_ids}")
+        retries_count += 1
+        if retries_count > retries:
+            raise AssertionError(
+                f"Discovered {len(event_worker_ids)} worker ids {event_worker_ids}, expected {NUM_MOCKERS}"
+            )
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)